packages/app/src/components/SourceForm.tsx

```diff
@@ -167,6 +167,19 @@ export function LogTableModelForm({
             connectionId={connectionId}
           />
         </FormRow>
+        <FormRow
```

Comment:
Can we move this to 'optional fields'?

Reply:
@wrn14897 sure, just update it
<img width="792" alt="image" src="https://github.com/user-attachments/assets/3e1759c5-d863-45e9-a1e9-958e65d1c996" />